pkg/stockings/stock.go

```diff
@@ -0,0 +1,195 @@
+package stockings
```

Comment:
so I think it could be possible in the very near future that we use multiple different stock apis to get prices and stuff  - for example, hungwei's portfolio has some dank EU stocks and idt iex can provide those. 

so with that context, I think we should move this to `pkg/stockings/iex.go`, and then also just have a generic `pkg/stockings/stock.go` which would contain non iex specific semantics.

Reply:
yeah the plan is to make stock.go into an interface and have the different apis implement these methods